### ISSUE TITLE
[question] LoadAdError: unable to resolve host, should be internal error (code 0) or network error (code 2)?

### ISSUE THREAD

**Issue by peterweb2005** [googleads/googleads-mobile-flutter#905]
as title

**Comment by huycozy** [comment]
Hi @peterweb2005 
When opening an issue, please provide information indicated in the issue template so that we can verify the issue properly, see [Bug report template](https://github.com/googleads/googleads-mobile-flutter/issues/new?assignees=&labels=&projects=&template=bug_report.md).

More specifically, please provide:
- A [completed and minimal reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example)
- The complete output of `flutter run -v` which contains entire stack trace of error.
- The current ads plugin version you are using
- Which environment did this issue happen in? (Android/iOS)

Regarding the error message above,  maybe the device does not have an internet connection, using proxy or on airplane mode. Please make sure ad request is not blocked by these reasons.

**Comment by peterweb2005** [comment]
so, for the error code 0, 
the party consider that is client error not network error?
which, if error code 0, should i ask user to retry the network, or check anything happend?
can any suggestions to handle error code 0 at runtime?

**Comment by huycozy** [comment]
For error code 0, it's probably because you used the wrong ID as I gave in [this issue](https://github.com/googleads/googleads-mobile-flutter/issues/897). 

For error code 2, it's probably related to network issue, you can ask user to retry to see if it happens again.

**Comment by peterweb2005** [comment]
but i what see:

LoadAdError(code: 0, domain: com.google.android.gms.ads, message: Error while connecting t
o ad server: Unable to resolve host "googleads.g.doubleclick.net": No address associated with hostname, responseInfo: ResponseInfo(responseId: null, mediationAdapterClassName: , adapterResp
onses: [], loadedAdapterResponseInfo: null), responseExtras: {})

**Comment by huycozy** [comment]
For that error, I would like to forward the same case from Developer Forum here: https://groups.google.com/g/google-admob-ads-sdk/c/-bDd5tNp0ns/m/6Skb8p33mzcJ

> One common reason for this is that your device is not able to contact a dns server in order to perform a lookup.
This most common reason for this is that your device doees not have a network connection.

This issue doesn't seem to be related to the google mobile ads flutter plugin, so I will close it from here. If you disagree, please write in comments with more info or reach out to our support team on the [Developer Forum](https://groups.google.com/forum/#!categories/google-admob-ads-sdk). Thanks!

**Comment by peterweb2005** [comment]
yes, you call that dns error, but it comes to error code 0, network error should be code 2

**Comment by peterweb2005** [comment]
i found that, in ios, the client network error will not state code 0, still code 2, LoadAdError(code: 2, domain: com.google.admob, message: The Internet connection appears to be offline
